src/SumatraStartup.cpp

```diff
@@ -796,7 +796,7 @@ int APIENTRY WinMain(_In_ HINSTANCE hInstance, _In_opt_ HINSTANCE hPrevInstance,
 
     retCode = RunMessageLoop();
     SafeCloseHandle(&hMutex);
-    CleanUpThumbnailCache(gFileHistory);

```

Comment:
It's a need for a better user experience. Some users - or even many - like to have such frequently list. (I saw this request before)